tools/packaging/kernel/configs/fragments/arm64/base.conf

```diff
@@ -42,3 +42,4 @@ CONFIG_ARM64_UAO=y
 
 CONFIG_NO_HZ_FULL=y
 CONFIG_GENERIC_MSI_IRQ_DOMAIN=y
+CONFIG_RANDOMIZE_BASE=y
```

Comment:
*spidey sense:* this may impact the performance, let's run the metrics CI

Reply:
@devimc I talked to @Amulyam24 about this one as well. It was in the base before, so this actually is only removing it for ppc64le. It's a 32-bit only option for Power atm, but seems to be fine to use by default on all other platforms (unless we're misunderstanding what the base conf file means of course).

Reply:
@clnperez metrics CI looks good